bin/run_functions.js

```diff
@@ -426,7 +426,7 @@ function executeProject(filename, project, options) {
                 }
                 let counter = 0;
                 async.whilst(
-                    function () {
+                    async function () {
```

Comment:
https://caolan.github.io/async/v3/docs.html#whilst - this change is due to the fact that whilst expects test to be an asynchronous function